components/ActivityIndicator/BUILD

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017-present The Material Components for iOS Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_ios_warnings//:strict_warnings_objc_library.bzl", "strict_warnings_objc_library")
+load("@build_bazel_rules_apple//apple:swift.bzl", "swift_library")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
+
+licenses(["notice"])  # Apache 2.0
+
+strict_warnings_objc_library(
+    name = "ActivityIndicator",
+    srcs = glob([
```

Comment:
we need to add the bundle: MaterialActivityIndicator.bundle
I think you need to make a filegroup and objc_bundle

filegroup(
    name = "BundleFiles",
    srcs = glob([
        "src/MaterialActivityIndicator.bundle/**",
    ]),
)

objc_bundle(
    name = "Bundle",
    bundle_imports = [":BundleFiles"],
)

Reply:
Done.